.github/workflows/ci-aarchxx.yml

```diff
@@ -75,6 +75,28 @@ jobs:
       env:
         DYNAMORIO_CROSS_AARCHXX_LINUX_ONLY: yes
 
+    - name: Send failure mail to dynamorio-devs
+      if: failure()
```

Comment:
Hmm, in the past we did have list emails on all PR failures.  I guess we can set up filters if we want to ignore PR failures (a reason to put `event_name` in the subject).  Or should we limit emails to cron jobs and master pushes?

Reply:
I've added a condition to send email only for failures on master branch (I assume the cron jobs should also satisfy that condition as they are also on the  master branch). That seems better than folks having to add email filters. Failures on master branch will probably be what others are mostly interested in. And the author of the PR gets emails directly from Github for CI triggered by them.